homepage/styles/main.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.navbar {
+  margin-top: 10px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
+  padding-bottom: 30px;
+  padding-top: 30px;
+  font-family: "Noto Sans", sans-serif;
```

Comment:
it looks like your `body` or `html` element should just start with "noto sans" as the font and then you can specify different kinds b/c you're using it so often.